<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TitanicChartsClass</class>
 <widget class="QMainWindow" name="Titanic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAffichage">
    <property name="title">
     <string>Affichage</string>
    </property>
    <addaction name="actionRatioHF"/>
    <addaction name="actionCategorie"/>
    <addaction name="actionAgeCategorie"/>
   </widget>
   <addaction name="menuAffichage"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRatioHF"/>
   <addaction name="actionCategorie"/>
   <addaction name="actionAgeCategorie"/>
  </widget>
  <action name="actionRatioHF">
   <property name="text">
    <string>Ratio H/F</string>
   </property>
  </action>
  <action name="actionCategorie">
   <property name="text">
    <string>Catégorie</string>
   </property>
  </action>
  <action name="actionAgeCategorie">
   <property name="text">
    <string>Âge/Catégorie</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
